src/Hazelcast.Net/Clustering/LoadBalancing/MembersInfoCollection.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using Hazelcast.Models;
+
+namespace Hazelcast.Clustering.LoadBalancing
+{
+    public class MembersInfoCollection
```

Comment:
Also can you add the proper `SuppressMessage` attribute in `CodeAnalysisSuppression.cs` so we don't get the silly warnings about the name ending in 'Collection'?

Reply:
I don't see any warning. Do you have it's code?

Reply:
This:

![image](https://user-images.githubusercontent.com/2846429/130044749-3711a0de-5d06-4c2f-bb50-f6829b1a524c.png)

You want to add:
```
[assembly: SuppressMessage(Analysis.Naming, Analysis.CA1711, Scope = Analysis.Scope.Type,
    Target = "~T:Hazelcast.Clustering.LoadBalancing.MemberInfoCollection",
    Justification = Analysis.Justification.CA1711_Reserved)]
```

around line 289 in `CodeAnalysisSuppression.cs` to fix this

Reply:
Done, please check